bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
Do we need this since we're not using yarn?

Reply:
Probably not, I'm happy to drop it.